README.txt

```diff
@@ -0,0 +1,237 @@
+# Project description
+
+Cloudstate is a specification, protocol, and reference implementation for providing distributed state management patterns suitable for **Serverless** computing. 
+The current supported and envisioned patterns include:
+
+* **Event Sourcing**
+* **Conflict-Free Replicated Data Types (CRDTs)**
+* **Key-Value storage**
+* **P2P messaging**
+* **CQRS read side projections**
+
+Cloudstate is polyglot, which means that services can be written in any language that supports gRPC, 
+and with language specific libraries provided that allow idiomatic use of the patterns in each language. 
+Cloudstate can be used either by itself, in combination with a Service Mesh, 
+or it is in envisioned that it will be integrated with other Serverless technologies such as [Knative](https://knative.dev/).
+
+Read more about the design, architecture, techniques, and technologies behind Cloudstate in [this section in the documentation](https://github.com/cloudstateio/cloudstate/blob/master/README.md#enter-cloudstate). 
+
+Cloudstate Python user language support is a library that implements the Cloudstate protocol and offers an elegant Python API 
```

Comment:
"offers an elegant Python API" is that a "pythonistic API" ?

Reply:
This is "bairrismo" 😆